### PR TITLE
Fix: use psr/container (is used from Symfony container) instead depre…

### PR DESCRIPTION
**Pull request by ondraondra81**
…cated container-interop/container-interop

### CODE REVIEW

**Comment by ondraondra81** [comment]
@leroy0211 can you check this please. Thank you

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     },
     "require-dev": {
         "symfony/yaml": "^3.2 || ^4.0",
-        "phpunit/phpunit": "^7.0"
+        "phpunit/phpunit": "~7.0"
```
Comment (leroy0211) [inline composer.json]: Could you explain why you changed the caret to a tilde? I don't see why phpunit should only be limited to 7.0.x

**Comment by leroy0211** [comment]
> @leroy0211 can you check this please. Thank you

Looks ok, only 1 minor question about your change. You'll see.

**Review by leroy0211 — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (ondraondra81) [reply composer.json]: I had a problem with version 8.x. I guess I forgot it.

**Review by leroy0211 — APPROVED** [review]

**Pull request merged**